src/imgix-api.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for imgix-management-js
+import { RequestInfo, RequestInit } from "@types/node-fetch";
+
+interface ImgixResponse {
+    data : {}
+    included: {}
+    jsonapi: {}
+    meta: {}
```

Comment:
I think this could be improved. At bare minimum the types need to be corrected for `data` and `included`. `data` can sometimes be an array, and `included` is an array, not an object.

Second stage would be type these as a `Json` type - it would need to be copied in since there is no default `Json` type.

Final stage would be to define a method and specific response type for each different API request.

Reply:
I thought the same thing but `typeof response.data` and `typeof response.included` both returned `Object`; unless that's totally the wrong way to check for that.

> Second stage would be type these as a Json type

Mind elaborating on this? 

> Final stage would be to define a method and specific response type for each different API request.

Yeah I'd like to work towards this eventually now that the API is stable.

Reply:
> thought the same thing but typeof response.data and typeof response.included both return Object; unless that's totally the wrong way to check for that.

`console.log(typeof []); //-> 'object'` 😬 I also double checked with the imgix documentation and they're definitely not objects.

> Mind elaborating on this?

Sure, throw me a bone 😄 

```ts
// Taken from https://github.com/microsoft/TypeScript/issues/1897#issuecomment-338650717 
type AnyJson = boolean | number | string | null | JsonArray | JsonMap;
interface JsonMap {  [key: string]: AnyJson; }
interface JsonArray extends Array<AnyJson> {}

interface ImgixResponse {
    data: JsonMap | JsonArray;
    included: JsonArray;
    jsonapi: JsonMap;
    meta: JsonMap;
}
```

Reply:
> In Javascript land, an "object" is an instance of a class. Therefore both {} and [] are instances of classes, so they are both "objects".

Oh boy, felt like I should have known that one already. Thanks for clearing that up 😅

Reply:
@frederickfogerty I went ahead with your suggestion to add json types. I was tempted to go further with more explicit types for `data`, `jsonapi`, and `meta`; but realized it was a bit of a rabbit hole that's not worth dealing with at the moment.